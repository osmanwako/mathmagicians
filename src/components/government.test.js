import React from 'react';
import renderer from 'react-test-renderer';
import Government from './government';

test('Government component renders correctly', () => {
    const tree = renderer.create(<Government />).toJSON();
    expect(tree).toMatchSnapshot();
});
