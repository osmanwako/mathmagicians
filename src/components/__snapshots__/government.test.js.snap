// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Government component renders correctly 1`] = `
<div
  className="desc-message"
>
  <p>
    Loading ...
  </p>
</div>
`;
